file-formats/uiad/type/zif_aff_uiad_v1.intf.abap

```diff
@@ -529,14 +529,40 @@ INTERFACE zif_aff_uiad_v1
 
     ty_tiles TYPE STANDARD TABLE OF ty_tile WITH DEFAULT KEY.
 
+  "! <p class="shorttext">Life Cycle Deprecation Status</p>
+  "! Life Cycle deprecation status
+  "! $values {@link zif_aff_uiad_v1.data:co_deprecate_status}
+  "! $default {@link zif_aff_uiad_v1.data:co_deprecate_status.active}
+  TYPES ty_deprecate_status TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Life Cycle Deprecation Status</p>
+    "! Life Cycle deprecation sStatus
+    BEGIN OF co_deprecate_status,
+
+      "! <p class="shorttext">Active</p>
+      "! LADI is active
+      active     TYPE ty_tile_type VALUE ' ',
+
+      "! <p class="shorttext">Deprecated</p>
+      "! LADI is deprecated
+      deprecated TYPE ty_tile_type VALUE 'D',
+
+      "! <p class="shorttext">Obsolete</p>
+      "! LADI is obsolete
+      obsolete   TYPE ty_tile_type VALUE 'O',
+
+    END OF co_deprecate_status.
+
+
   TYPES:
-    "! <p class="shorttext">Lifecycle</p>
-    "! Lifecycle state
+    "! <p class="shorttext">Life Cycle Deprecation</p>
+    "! Life cycle deprecation
     BEGIN OF ty_lifecycle,
```

Comment:
Should it be life_cycle since above it is written as 2 different words?
Lifecycle as just one word is also be possible, both possibilities are valid (just googled it), but titles, description and variable names should be consistent